### PR TITLE
Remove unused Sylius Labs polyfill dependencies

### PR DESCRIPTION
**Pull request by stloyd**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | yes
| Deprecations?   | cleaning them
| Related tickets | refs https://github.com/SyliusLabs/PolyfillSymfonyEventDispatcher/issues/3
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/OrderBundle/Command/RemoveExpiredCartsCommand.php [hunk 1]
```diff
@@ -31,15 +35,16 @@ protected function configure(): void
...
             'Command will remove carts that have been idle for <info>%s</info>.',
             (string) $expirationTime
         ));
 
-        $expiredCartsRemover = $this->getContainer()->get('sylius.expired_carts_remover');
+        /** @var ExpiredCartsRemoverInterface $expiredCartsRemover */
+        $expiredCartsRemover = $this->container->get('sylius.expired_carts_remover');
         $expiredCartsRemover->remove();
 
         return 0;
```
Comment (Roshyo) [inline src/Sylius/Bundle/OrderBundle/Command/RemoveExpiredCartsCommand.php:50]: While we're here, we should use constants for this `Command::SUCCESS`

Comment (loic425) [reply src/Sylius/Bundle/OrderBundle/Command/RemoveExpiredCartsCommand.php:50]: @Roshyo I think a dedicated PR for this one should be better.

Comment (stloyd) [reply src/Sylius/Bundle/OrderBundle/Command/RemoveExpiredCartsCommand.php:50]: I agree about new PR doing that, also IMHO it's just "syntax sugar" that doesn't give any advantage.

**Review by Roshyo — APPROVED** [review]

File: config/bundles.php [hunk 2]
```diff
@@ -59,5 +59,4 @@
     SyliusLabs\DoctrineMigrationsExtraBundle\SyliusLabsDoctrineMigrationsExtraBundle::class => ['all' => true],
     Sonata\Doctrine\Bridge\Symfony\SonataDoctrineSymfonyBundle::class => ['all' => true],
     Sonata\Twig\Bridge\Symfony\SonataTwigSymfonyBundle::class => ['all' => true],
-    SyliusLabs\Polyfill\Symfony\Security\Bundle\SyliusLabsPolyfillSymfonySecurityBundle::class => ['all' => true],
```
Comment (GSadee) [inline config/bundles.php:62]: Could you add a note to an UPGRADE file?

File: src/Sylius/Bundle/CoreBundle/Command/InstallCommand.php [hunk 3]
```diff
@@ -67,7 +67,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $outputStyle->writeln('<info>Installing Sylius...</info>');
         $outputStyle->writeln($this->getSyliusLogo());
 
-        $this->ensureDirectoryExistsAndIsWritable((string) $this->getContainer()->getParameter('kernel.cache_dir'), $output);
+        $this->ensureDirectoryExistsAndIsWritable((string) $this->container->getParameter('kernel.cache_dir'), $output);
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/Command/InstallCommand.php:70]: Looking at the tests, it looks like it's not working after the refactor, will you try to fix the problem?

**Pull request closed without merging**